tasty/HasCal/Test/HourClock.hs

```diff
@@ -0,0 +1,75 @@
+{-| HourClock
+    [example](https://github.com/tlaplus/Examples/blob/master/specifications/SpecifyingSystems/HourClock/HourClock.tla)
+    from figure 2.1 on page 20 in Lamport's *Specifying Systems* book:
+
+@
+        VARIABLE hr
+
+        HCini == hr \in (1 .. 12)
+        HCnxt == hr' = IF hr # 12 THEN hr + 1 ELSE 1
+        HC    == HCini /\ [][HCnxt]_hr
```

Comment:
I suggest using leading `> ` for TLA+ code blocks, mainly because you don't risk special characters like `\` or `"` being interpreted by the haddock generator.  I usually only use `@` code blocks for Haskell code because they permit linking to modules and identifiers.  Also, `@` interacts poorly with respect to indentation; if you do use `@` then you typically want to dedent the block to the first column, otherwise it will render weird in the haddocks.